.github/workflows/ci.yml

```diff
@@ -126,7 +126,7 @@ jobs:
 
     - name: Configure CMake
       shell: bash
-      run: cmake -S $GITHUB_WORKSPACE -B $GITHUB_WORKSPACE/build -DCMAKE_INSTALL_PREFIX=$GITHUB_WORKSPACE/install -DSFML_BUILD_EXAMPLES=TRUE -DCMAKE_VERBOSE_MAKEFILE=ON -DSFML_BUILD_TEST_SUITE=TRUE ${{matrix.platform.flags}} ${{matrix.config.flags}} ${{matrix.type.flags}}
+      run: cmake --preset dev -DCMAKE_VERBOSE_MAKEFILE=TRUE ${{matrix.platform.flags}} ${{matrix.config.flags}} ${{matrix.type.flags}}
```

Comment:
Did you intentionally change from `-DCMAKE_VERBOSE_MAKEFILE=ON` to `-DCMAKE_VERBOSE_MAKEFILE=TRUE`?

Reply:
Probably just an incidental thing. I'll use whichever style you prefer.

Reply:
I think I'd just go with ON/OFF

Reply:
Okay I changed it back to using `ON`